### PR TITLE
Add edX video ID section

### PR DESCRIPTION
**Pull request by lamagnifica**
Added the video_course.rst file to describe what course teams do after automated processing is complete.
@nasthagiri, @frrrances, @WatsonEmily , @lou-wang, @srpearce 
@mhoeber, @catong FYI.

### CODE REVIEW

**Comment by lamagnifica** [comment]
An updated HTML version of all doc to date is available at http://draft-processing-video-files.readthedocs.org/en/latest/video_course.html

@nasthagiri, @frrrances, @WatsonEmily , @lou-wang, @srpearce 
@mhoeber, @catong FYI.

File: en_us/video/source/video_course.rst [hunk 1]
```diff
@@ -4,5 +4,101 @@
...
+************************************************
+Add the edX Video ID to a Video Component
+************************************************
+
+This section describes the process that you follow in place of the `Step 4.
+Create a Video Component`_ section in the *Building and Running an edX Course*
+guide. This section assumes that you are familiar with the process described in
+the `Developing Your Course`_ chapter.
+
+You complete these steps in Studio on the Course Outline page: from the
```
Comment (catong) [inline en_us/video/source/video_course.rst]: Make step 1 "From the **Content**menu select..."?

File: en_us/video/source/video_course.rst [hunk 2]
```diff
@@ -4,5 +4,101 @@
...
+**Content** menu select **Outline**.
+
+#. Select or add a unit, and then click **Video** to add a video component. To
```
Comment (catong) [inline en_us/video/source/video_course.rst]: no need for "and" before "then"

File: en_us/video/source/video_course.rst [hunk 3]
```diff
@@ -4,5 +4,101 @@
...
+   edit an existing video component, locate the video component window and then
```
Comment (catong) [inline en_us/video/source/video_course.rst]: no need for "and" before "then"

File: en_us/video/source/video_course.rst [hunk 4]
```diff
@@ -4,5 +4,101 @@
...
+   click **Edit**. The Editing: Video popup opens to the Basic tab.
+   
+#. In the **Component Display Name** field, enter the identifying name that you
+   want students to see for this video.
+
+#. Delete the value in the **Default Video URL** field. This field must be
+   blank.
+
+#. To supply captions for the video, you select one of the options for the
```
Comment (catong) [inline en_us/video/source/video_course.rst]: Is "captions" also our term for the transcript? I wonder if that will cause confusion with in-screen captioning.

File: en_us/video/source/video_course.rst [hunk 5]
```diff
@@ -4,5 +4,101 @@
...
+   **Default Timed Transcript** field.
+
+.. Rachel is that ^ correct, or should they always use the Upload New Transcript option (and not the Import YouTube Transcript option)?
+
+   Captions are required for all videos. See `Step 2. Create or Obtain a Video
```
Comment (catong) [inline en_us/video/source/video_course.rst]: Same question about "Captions" as above

**Comment by WatsonEmily** [comment]
@uhanile - please review and provide feedback. Thanks!

**Comment by lamagnifica** [comment]
and @rjgrubb!

File: en_us/video/source/video_course.rst [hunk 6]
```diff
@@ -4,5 +4,101 @@
...
+
+************************
+Copy the edX Video ID
+************************
+ 
+#. Open the course in Studio. 
+
+#. Select **Content**, then **Video Uploads**.
+
+#. In the Previous Uploads list, locate the video that you want to include in
```
Comment (srpearce) [inline en_us/video/source/video_course.rst]: Bold "Previous Uploads"

File: en_us/video/source/video_course.rst [hunk 7]
```diff
@@ -4,5 +4,101 @@
...
+   the course.
+
+#. Select the value in the **Video ID** column for the video. A value appears
+   in this column only after video processing is complete and successful for
+   the video.
+
+#. Right-click and select **Copy**. You can then paste this value in to the
```
Comment (srpearce) [inline en_us/video/source/video_course.rst]: I'd reiterate: "Right-click the value and select **Copy**"

Also, either "into" or just "in"

File: en_us/video/source/video_course.rst [hunk 8]
```diff
@@ -4,5 +4,101 @@
...
+Add the edX Video ID to a Video Component
+************************************************
+
+This section describes the process that you follow in place of the `Step 4.
+Create a Video Component`_ section in the *Building and Running an edX Course*
+guide. This section assumes that you are familiar with the process described in
+the `Developing Your Course`_ chapter.
+
+You complete these steps in Studio on the Course Outline page: from the
+**Content** menu select **Outline**.
```
Comment (srpearce) [inline en_us/video/source/video_course.rst]: Comma after "menu"

File: en_us/video/source/video_course.rst (hunk 2 above)
Comment (srpearce) [inline en_us/video/source/video_course.rst]: I'd respectfully disagree - without "and" it becomes a comma splice.

File: en_us/video/source/video_course.rst (hunk 3 above)
Comment (srpearce) [inline en_us/video/source/video_course.rst]: I respectfully disagree here too - we don't want to say "locate the video component window then click **Edit**." I'd just add a comma after "window."

File: en_us/video/source/video_course.rst [hunk 9]
```diff
@@ -4,5 +4,101 @@
...
+
+#. Select or add a unit, and then click **Video** to add a video component. To
+   edit an existing video component, locate the video component window and then
+   click **Edit**. The Editing: Video popup opens to the Basic tab.
```
Comment (srpearce) [inline en_us/video/source/video_course.rst]: Bold UI elements (pop-up name, tab name)

File: en_us/video/source/video_course.rst [hunk 10]
```diff
@@ -4,5 +4,101 @@
...
+   
+#. In the **Component Display Name** field, enter the identifying name that you
+   want students to see for this video.
+
+#. Delete the value in the **Default Video URL** field. This field must be
+   blank.
+
+#. To supply captions for the video, you select one of the options for the
+   **Default Timed Transcript** field.
```
Comment (srpearce) [inline en_us/video/source/video_course.rst]: This should be an imperative sentence - delete "you" before "select"

File: en_us/video/source/video_course.rst [hunk 11]
```diff
@@ -4,5 +4,101 @@
...
+
+.. Rachel is that ^ correct, or should they always use the Upload New Transcript option (and not the Import YouTube Transcript option)?
+
+   Captions are required for all videos. See `Step 2. Create or Obtain a Video
+   Transcript`_ in the *Building and Running an edX Course* guide.
+
+5. At the top of the Editing: Video popup click **Advanced**. Additional fields
```
Comment (srpearce) [inline en_us/video/source/video_course.rst]: **Editing: Video**

File: en_us/video/source/video_course.rst [hunk 12]
```diff
@@ -4,5 +4,101 @@
...
+
+#. Complete the **Download Transcript Allowed** and **Downloadable Transcript
+   URL** fields.
+
+#. In the **EdX Video ID** field, paste the ID of the video file that you want
+   to play. See :ref:`Copy the edX Video ID`.
+   
+#. Complete the **Show Transcript** and **Transcript Languages** fields.
+
+#. Optionally, in the **Upload Handout** field include an accompanying file in
```
Comment (srpearce) [inline en_us/video/source/video_course.rst]: Comma after "field"

File: en_us/video/source/video_course.rst [hunk 13]
```diff
@@ -4,5 +4,101 @@
...
+
+#. Set the **Video Download Allowed** field to **True** or **False** to define
+   whether students can download this video.
+
+.. Rachel is that ^ correct? should they always select True?
+
+11. Do not supply any values for the **Video File URLs** field. This field must
+    be blank.
+
+#. Optionally, supply a **Video Start Time** or a **Video Stop Time** if you
```
Comment (srpearce) [inline en_us/video/source/video_course.rst]: Either "supply a video start time or a video stop time" or "Supply values in the **Video Start Time** or **Video Stop Time** fields"

**Comment by srpearce** [comment]
Preliminary editing pass complete.

File: en_us/video/source/video_course.rst (hunk 7 above)
Comment (nasthagiri) [inline en_us/video/source/video_course.rst]: I'd like to make sure they select the entire video-ID value first, before they right-click to Copy.

File: en_us/video/source/video_course.rst [hunk 14]
```diff
@@ -4,5 +4,101 @@
...
+
+.. _Add the edX Video ID to a Video Component:
+
+************************************************
+Add the edX Video ID to a Video Component
+************************************************
+
+This section describes the process that you follow in place of the `Step 4.
+Create a Video Component`_ section in the *Building and Running an edX Course*
+guide. This section assumes that you are familiar with the process described in
```
Comment (nasthagiri) [inline en_us/video/source/video_course.rst]: Is there a back-reference from the `Create a Video Component` section in `Building and Running an edX Course` guide back to here?

File: en_us/video/source/video_course.rst [hunk 15]
```diff
@@ -4,5 +4,101 @@
...
+
+.. Rachel is that ^ correct? should they always select True?
+
+11. Do not supply any values for the **Video File URLs** field. This field must
+    be blank.
+
+#. Optionally, supply a **Video Start Time** or a **Video Stop Time** if you
+   do not want the entire video to play.
+
+#. Do not supply any values for the **YouTube ID** speed fields. These fields
```
Comment (nasthagiri) [inline en_us/video/source/video_course.rst]: Perhaps Steps 11 and 13 should not be steps, but highlighted Notes?  Since they are not additional steps to follow, but rather things to NOT do.

File: en_us/video/source/video_course.rst (hunk 9 above)
Comment (uhanile) [inline en_us/video/source/video_course.rst]: This line was confusing to me. I read it as "The Editing: /  Video popup opens to the Basic tab." Perhaps say "A popup titled "Editing: Video" opens to the Basic tab"

File: en_us/video/source/video_course.rst (hunk 10 above)
Comment (nasthagiri) [inline en_us/video/source/video_course.rst]: This doesn't work for me. That is, I don't see any options for the Default Timed Transcript field after deleting the Default Video URL.

File: en_us/video/source/video_course.rst [hunk 16]
```diff
@@ -4,5 +4,101 @@
...
+
+   Captions are required for all videos. See `Step 2. Create or Obtain a Video
+   Transcript`_ in the *Building and Running an edX Course* guide.
+
+5. At the top of the Editing: Video popup click **Advanced**. Additional fields
+   appear below the **Component Display Name** and **Default Timed Transcript**
+   fields.
+
+#. Complete the **Download Transcript Allowed** and **Downloadable Transcript
+   URL** fields.
```
Comment (nasthagiri) [inline en_us/video/source/video_course.rst]: I'm not sure we really want course authors to set the `Downloadable Transcript URL` field if edx-video-ids are used.

**Comment by nasthagiri** [comment]
@lamagnifica  Perhaps someone from the Kiev team (who is familiar with the Video module) can also review this.

**Comment by lamagnifica** [comment]
@tymofij and @polesye and @olmar , your insights welcome!

File: en_us/video/source/video_course.rst [hunk 17]
```diff
@@ -4,5 +4,101 @@
...
+#. In the **Component Display Name** field, enter the identifying name that you
+   want students to see for this video.
+
+#. Delete the value in the **Default Video URL** field. This field must be
+   blank.
+
+#. To supply captions for the video, you select one of the options for the
+   **Default Timed Transcript** field.
+
+.. Rachel is that ^ correct, or should they always use the Upload New Transcript option (and not the Import YouTube Transcript option)?
```
Comment (rjgrubb) [inline en_us/video/source/video_course.rst]: They can use either method, Upload a New Transcript (which is what our 3Play-using members, like Harvard, will use) or Import YouTube Transcript. Overall, this initial draft is hard to follow. Separately, I sent Alison a revised draft that breaks the options into smaller sections to simplify the instructions.

File: en_us/video/source/video_course.rst (hunk 14 above)
Comment (lamagnifica) [inline en_us/video/source/video_course.rst]: No, and there will not be, Nimisha. This document is intended for use by course teams at member institutions only. Including a link to it in the completely open Building & Running guide would defeat the purpose.

**Comment by lamagnifica** [comment]
@rjgrubb , @lou-wang, @tymofij, @polesye, @olmar, @remyedX, and @uhanile this has been reworked significantly to reduce content to only what is different about the process for course teams using the new workflow.
@nasthagiri, @frrrances, @WatsonEmily , @srpearce , @mhoeber, @catong FYI.

File: en_us/video/source/video_course.rst [hunk 18]
```diff
@@ -4,5 +4,84 @@
...
+
+   When you supply a valid edX video ID in this field, it overrides values in
+   the **Default Video URL**, **Video File URLs**, and the **YouTube ID**
+   fields.
+
+#. Set the **Video Download Allowed** field to **True** or **False** to define
+   whether students can download this video.
+
+#. Click **Save**. The referenced video appears in the video component.
+
```
Comment (rjgrubb) [inline en_us/video/source/video_course.rst:80]: Can we add two notes at the end of 4.2:

NOTE: By adding the edX ID, you associate your video component with the following elements: YouTube, AWS, and mobile. Therefore, you do not need to add a separate element for YouTube or download links in Studio. By adding the edX ID, these steps are complete. 

NOTE: Adding the edX ID is separate from adding transcripts for your files. You still need to add transcripts for your videos in Studio.

Comment (lamagnifica) [inline en_us/video/source/video_course.rst:80]: @rjgrubb, I rewrote the information on lines 69-71 to include the info from your first suggested note. 
I prefaced the paragraph that starts immediately below on line 78 with the information in your second note.
Thanks!

Comment (rjgrubb) [inline en_us/video/source/video_course.rst:80]: Thanks!

**Comment by lamagnifica** [comment]
Closing this PR. Replacing with a new PR.
